electrum/gui/qt/main_window.py

```diff
@@ -258,7 +258,7 @@ def add_optional_tab(tabs, tab, icon, description):
         QShortcut(QKeySequence("Ctrl+PgDown"), self, lambda: wrtabs.setCurrentIndex((wrtabs.currentIndex() + 1)%wrtabs.count()))
 
         for i in range(wrtabs.count()):
-            QShortcut(QKeySequence("Alt+" + str(i + 1)), self, lambda i=i: wrtabs.setCurrentIndex(i))
+            QShortcut(QKeySequence("Alt+" + str(i + 1)), self, lambda i: wrtabs.setCurrentIndex(i))
```

Comment:
Looks unrelated to coldcard stuff?

More importantly it is incorrect. The existing code here works and this change breaks it.
(Interestingly PyCharm marks the old code as an error, but its static analyser is just not good enough I guess.)

Reply:
Hm, I thought is was caused by me hitting a key with the cursor in the wrong spot.